src/app/AppDom.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import MenuDom from '../menu';
+import BodyDom from "../body";
+import {Page} from "../models";
+
+export default class AppDom extends React.Component<Page> {
+    constructor(page:Page){
```

Comment:
The common pattern is to write `props` instead of `page`
```
constructor(props: Page) {
   super(props);
   ...
}
```

Reply:
removed